modules/icgc-argo-workflows/prep/metrics/resources/usr/bin/rnaaln.py

```diff
@@ -29,17 +29,15 @@
 tool_fieldmap = { # 'target name' : 'original name'
     'hisat2': {
         'paired_total' : 'paired_total',
-        'unpaired_total' : 'unpaired_total',
-        'overall_alignment_rate' : 'overall_alignment_rate'
+        'unpaired_total' : 'unpaired_total'
     },
     'star' : {
         'total_reads' : 'total_reads',
         'avg_input_read_length' : 'avg_input_read_length',
-        'uniquely_mapped_percent' : 'uniquely_mapped_percent',
+        'pct_uniquely_mapped' : 'uniquely_mapped_percent',
         'avg_mapped_read_length' : 'avg_mapped_read_length',
         'num_splices' : 'num_splices',
         'num_annotated_splices' : 'num_annotated_splices',
-        'mismatch_rate' : 'mismatch_rate',
```

Comment:
`mismatch_rate` in `star` is duplicated to `mismatch_bases_rate` in `samtools_stats`. So removing it.